src/wallet/coinselection.h

```diff
@@ -139,9 +139,9 @@ struct CoinSelectionParams {
     /** Randomness to use in the context of coin selection. */
     FastRandomContext& rng_fast;
     /** Size of a change output in bytes, determined by the output type. */
-    size_t change_output_size = 0;
+    int change_output_size = 0;
```

Comment:
It's simply just to ensure consistency in transaction size and weight calculation within the wallet and prevents conversion overflow when calculating `max_selection_weight` as it happened here https://cirrus-ci.com/task/6341256662482944

---
I think it definitely will be worth it to define a datatype for size in the entire codebase.
But I am limiting this change to affects only the wallet size variables `change_output_size`, `change_spend_size` and `tx_noinputs_size`, because they relate to this PR.